src/reference/02-DetailTopics/01-Using-sbt/03-Cross-Build.md

```diff
@@ -308,6 +308,32 @@ A custom function is mainly used when cross-building and a dependency
 isn't available for all Scala versions or it uses a different convention
 than the default.
 
+#### Conditional sbt statements depending on binary version
+
+Sometimes you might need to change your sbt settings at runtime, depending on which binary
```

Comment:
This statement doesn't make sense. What setting is being changed at what program's runtime?